torch/nn/modules/activation.py

```diff
@@ -886,6 +886,23 @@ def extra_repr(self) -> str:
         return str(self.lambd)
 
 
+def _arg_cuda_or_cpu(x: Optional[torch.Tensor]) -> bool:
+  if x is None:
+     return True
+  else:
+    return x.is_cuda or 'cpu' in str(x.device)
+
+  return False
+
+def _arg_requires_grad(x: Optional[torch.Tensor]) -> bool:
+    if x is None:
+        return False
+    else:
+        return x.requires_grad
+
+    return True
```

Comment:
this seems unreachable, right?

Reply:
In principle, unless you ever run into a type checker that checks implicit returns regardless of reachability.  Putting the conditional into the else path is required to get type refinement for static torchscript typing.